src/arch/arm/armv/armv8-a/32/machine_asm.S

```diff
@@ -7,9 +7,9 @@
 #include <config.h>
 #include <machine/assembler.h>
 
-#if defined(CONFIG_ARM_CORTEX_A53)
+#if defined(CONFIG_ARM_CORTEX_A53) || defined(CONFIG_ARM_CORTEX_A72)
 
-/* A53 hardware does not support TLB locking */
+/* A53,A72 hardware does not support TLB locking */
```

Comment:
I wonder, does ARMv8 support this for AARCH32 at all?

Reply:
It's implementation defined:
> VMSAv8-32 CP15 c10 register summary
The CP15 c10 registers provide:
• Virtual memory control registers. That is, they provide registers from the functional group described in Virtual memory control registers, functional group on page G4-4233.
• Reserved encodings for IMPLEMENTATION DEFINED TLB control functions, including lockdown.